### PR TITLE
Kops release 1.21

### PR DESCRIPTION
**Pull request by olemarkus**
/cc @rifelpet @hakman

### CODE REVIEW

**Comment by rifelpet** [comment]
All of these changes look good to me, thanks!

/lgtm
/approve

**Comment by hakman** [comment]
@olemarkus can you rebase this, please?

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -336,7 +336,8 @@ milestone_applier:
     main: v1.21
     master: v1.21
   kubernetes/kops:
-    master: v1.20
+    master: v1.21
+    release-1.19: v1.20
```
Comment (cblecker) [inline config/prow/plugins.yaml:340]: I think this is incorrect. There are two 1.19 branches

**Review by cblecker — CHANGES_REQUESTED** [review]

File: config/prow/plugins.yaml (hunk 1 above)
Comment (olemarkus) [reply config/prow/plugins.yaml]: Thanks. That is a very good catch.

**Comment by hakman** [comment]
@olemarkus I merged the pieces related to jobs this morning so this PR should now only about adding the milestones. Could you rebase?

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

**Comment by hakman** [comment]
Thanks @spiffxp @cblecker 😄

**Pull request merged**